spec/models/document_spec.rb

```diff
@@ -5,4 +5,63 @@
 RSpec.describe Document, type: :model do
   ## Test of relationships
   it { should belong_to(:account) }
+
+  context 'Document creation' do
+    let!(:current_account) { create(:account) }
+    let(:create_document) { create :document, account: current_account }
+
+    context 'when it is first document' do
+      it 'adds new document label' do
+        expect(current_account.labels).to be_empty
+        create_document
```

Comment:
Suggest to do `expect { create_document }.to change { current_account.labels.count }.from(0).to(1)`
probably some `reload` needed